docs/how-to/upgrade.md

```diff
@@ -9,10 +13,12 @@ Migrating data from Prometheus to Mimir or from one charm revision of
 Loki to another is complex and nuanced. At this point, we recommend a 
 retention-based phase-out.
 
-## Migration via retention-based phase-out
+#### Migration via retention-based phase-out
 
 1. Deploy COS in a separate model next to COS Lite
 2. Relate the new COS charms to the same applications COS Lite is related to.
 3. Wait for the retention period to elapse for COS Lite.
 4. Verify the same data is available both in COS Lite and in COS
-5. Decommission COS Lite.
\ No newline at end of file
+5. Decommission COS Lite.
+
+### Migrate from COS Lite 1 to COS Lite 2
```

Comment:
Looks like there's something missing here. (I know it's a draft but point out just in case:))

Reply:
@MichaelThamm @Abuelodelanada we may need to update this section if something comes up in the itests,